src/Jackett.Common/Indexers/HDOlimpo.cs

```diff
@@ -118,33 +121,35 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
             bodyParts.Add("--" + boundary + "--");
             var body = string.Join("\r\n", bodyParts);
 
-            var headers = new Dictionary<string, string>();
-            headers.Add("Content-Type", "multipart/form-data; boundary=" + boundary);
-            addXsrfTokenHeader(headers, configData.CookieHeader.Value);
+            var headers = new Dictionary<string, string>
+            {
+                {"Content-Type", "multipart/form-data; boundary=" + boundary}
```

Comment:
This feels like they should be two separate header entries. Is there a reason why they aren't?

Reply:
It's ok, I'm using this tracker. Take a look at => https://stackoverflow.com/a/30194096
There are external libraries to do this more elegantly but I don't want to add one just for this case.